services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/PathSegment.java

```diff
@@ -78,6 +78,8 @@
 
     private Long latency;
 
+    private boolean failed = false;
```

Comment:
Shouldn't we add it to constructor, so the copying builder copies the value? Also, shouldn't it be a enum like SegmentStatus?